src/components/PageNotFound.js

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import { styled, ThemeProvider, createTheme } from '@mui/material/styles';
+import { Typography } from '@mui/material';
+import ArrowBackIcon from '@mui/icons-material/ArrowBack';
+import NavigateButton from './layout/NavigateButton/NavigateButton';
+
+import imgPageNotFound from '../images/page-not-found.jpg';
+
+const theme = createTheme({
+  palette: {
+    background: {
+      default: '#ffffff',
+      paper: '#eae9e9',
+    },
+    text: {
+      primary: '#000000',
+    },
+    shadow: {
+      boxShadow: `
+  0px 0px 0px 0.5px rgba(50, 50, 93, 0.1),
+  0px 2px 5px 0px rgba(50, 50, 93, 0.1),
+  0px 1px 1.5px 0px rgba(0, 0, 0, 0.07)
+`,
+    },
+  },
+});
+
+const NotFoundImage = styled('img')({
+  width: '100%',
+  height: 'auto',
+  objectFit: 'cover',
+});
+
+const NotFoundBody = styled('div')({
+  backgroundColor: theme.palette.background.default,
+  marginTop: '1rem',
+});
+const NotFoundPageContainer = styled('div')({
+  display: 'flex',
+  justifyContent: 'center',
+  alignItems: 'center',
+  minHeight: 'calc(100vh - 4rem)', // header with a height of 4rem;
+});
+
+const NotFoundWrapper = styled('div')({
+  backgroundColor: theme.palette.background.paper,
+  maxWidth: '37.5rem',
+  borderRadius: theme.spacing(1.5),
+  padding: theme.spacing(2),
+  margin: theme.spacing(2),
+  boxShadow: theme.palette.shadow.boxShadow,
+});
+
+function PageNotFound() {
+  return (
+    <ThemeProvider theme={theme}>
+      <NotFoundBody>
+        <Typography
+          variant="h4"
+          style={{
+            color: theme.palette.text.primary,
+            display: 'flex',
+            justifyContent: 'center',
+          }}
+        >
+          Page not found
+        </Typography>
```

Comment:
I'm thinking we can remove this text; since the image itself already does say the same thing. (And if the image doesn't load for any reason; the alt text will be shown already). 

![](https://github.com/Code-the-Dream-School/dd-prac-team4-front/assets/9094098/c8e09c2c-65d0-4973-a679-a94916fec01a)

Reply:
Done